### PR TITLE
Adds links to share on x, facebook, linkedin.

### LINKED ISSUE #46: Add link per each post to share it on twitter / fb / linkedin

### PR DESCRIPTION
**Pull request by AucT**
Closes #46

## Summary by CodeRabbit

- **New Features**
	- Introduced a social sharing section in blog posts, allowing users to easily share content on X, Facebook, and LinkedIn.
	- Added new SVG icons for Facebook, LinkedIn, and "X" to enhance social media integration and accessibility in the user interface.
	- Implemented a dynamic SVG rendering template for improved icon management.
	- Enhanced the layout and styling of individual blog posts for better readability and user experience.

- **Bug Fixes**
	- Improved accessibility features for new icons to ensure compatibility with assistive technologies.

### CODE REVIEW

File: themes/beaver/layouts/partials/icons/facebook.html
Comment (pftg) [inline themes/beaver/layouts/partials/icons/facebook.html:1]: Why does it have HTML ext and not SVG?

File: themes/beaver/layouts/partials/icons/facebook.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg"
+aria-label="Facebook" role="img"
+viewBox="0 0 512 512"><path
+d="m0 0H512V512H0"
+fill="#fff"/><path d="m331 312h-45v134a192 192 0 10-60 0V312h-49v-56h49v-45q4-89 116-68v47h-26q-29 1-30 30v36h53.5" fill="#1877f2"/></svg>
```
Comment (pftg) [inline themes/beaver/layouts/partials/icons/facebook.html]: Could you add empty new line

File: themes/beaver/layouts/blog/single.html [hunk 2]
```diff
@@ -30,6 +30,43 @@ <h1 class="heading">{{ .Title }}</h1>
...
+              display: inline-block;
+              width: 36px;
+              height: 36px;
+            }
+          
+            .social-share svg {
+              width: 100%;
+              height: 100%;
+            }
+          </style>
```
Comment (pftg) [inline themes/beaver/layouts/blog/single.html]: move to seprate css file

File: themes/beaver/layouts/partials/icons/facebook.html (hunk 1 above)
Comment (AucT) [reply themes/beaver/layouts/partials/icons/facebook.html]: No need. Btw I converted to svg and use svg.html partial so you can preview svg in static/img/icons in your editor.

File: themes/beaver/layouts/blog/single.html (hunk 2 above)
Comment (AucT) [reply themes/beaver/layouts/blog/single.html]: Moved.

**Comment by AucT** [comment]
preview:
XdGo5hH3wS

File: themes/beaver/layouts/partials/icons/facebook.html
Comment (pftg) [reply themes/beaver/layouts/partials/icons/facebook.html:1]: @AucT please add more reasons we are breaking consistency? Such names makes confusions for developers, so we need real good reasons for this

**Review by pftg — APPROVED** [review]
Let's follow default style to provide empty line at the end.

**Pull request merged**